ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -1851,6 +1851,7 @@ def dtype_array_query(
             max_size=len(shape),
         )
     )
+
```

Comment:
Hello @marschattha, can you kindly please revert this line here? this update seems irrelevant to the PR. Thanks in advance!